### ISSUE TITLE
Show space-indented paragraphs in list items as fully indented

### ISSUE THREAD

**Issue by hkdobrev** [careteditor/caret#51]
Basically imitate what Sublime Text + Markdown Extended package does.

When you have a list of items with paragraphs, e.g.:

``` md
- Very important item
  Explaining why this is important
  on multiple lines

  or multiple paragraphs

- Another very important item
  And more explaining
```

If you put just two spaces after the first newline in a list item, the whole paragraph gets indented like above. In Caret though you see the literal indentation:

``` md
- Very important item
  Explaining why this is important
on multiple lines

or multiple paragraphs

- Another very important item
  And more explaining
```

With Sublime Text + Markdown Extended you get all paragraphs in an item indented visually with the spaces you've placed the first time, but it doesn't even put spaces in the text, just visually shows it like that and I really like the looks of it while I'm writing.

**Comment by erusev** [comment]
r u sure that it aligns to text and not the dash as in:

```
   - Very important item
   Explaining why this is important
   on multiple lines
```

**Comment by hkdobrev** [comment]
@erusev What I meant is it doesn't do it by default, but if you put two spaces after the newline it respects for any wrapped line or paragraph below.

**Comment by erusev** [comment]
@hkdobrev that's true, we are thinking about incorporating sth like this

**Comment by astoilkov** [comment]
This is now implemented in the latest 1.4.0 release.
